python/README.md

```diff
@@ -0,0 +1,115 @@
+# Point One FusionEngine Python Client
+
+This directory includes Python support for the Point One FusionEngine message protocol, along with analysis tools,
+data extraction utilities, and example usage scripts.
+
+See https://github.com/PointOneNav/fusion-engine-client for full details. See http://docs.pointonenav.com/fusion-engine/
+for the latest API documentation.
+
+#### Python Support
+- Python 3.4 or later
+
+### Directory Structure
+  - `python/` - Python source files
+    - `bin/` - Application files
+      - [p1_display.py](examples/p1_display.py) - Generate HTML plots of vehicle trajectory, etc. (see also
+        `analyze_data.py`)
+    - `examples/` - Python example applications
+      - [analyze_data.py](examples/analyze_data.py) - Generate HTML plots of vehicle trajectory, INS filter state, etc.
+        - This class is used by the `bin/p1_display.py` application
+      - [extract_imu_data.py](examples/extract_imu_data.py) - Generate a CSV file containing recorded IMU measurements
+      - [extract_position_data.py](examples/extract_position_data.py) - Generate CSV and KML files detailing the vehicle
+        position over time
+        - This script also includes an example of time-aligning multiple message types
+      - [extract_satellite_info.py](examples/extract_satellite_info.py) - Generate a CSV file containing satellite 
+        azimuth/elevation and C/N0 information over time
+      - [message_decode.py](examples/message_decode.py) - Read a `.p1log` binary file and decode the contents
+      - [tcp_client.py](examples/tcp_client.py) - Connect to a device over TCP and decode/display messages in real time
+      - [udp_client.py](examples/udp_client.py) - Connect to a device over UDP and decode/display messages in real time
+    - `fusion_engine_client` - Top Python package directory
+      - `messages` - Python message definitions
+    
+### Usage Instructions
+
+To use the Python library:
+
+1. Install Python 3.4 (or later) and pip.
+2. Install the Python requirements:
+   ```bash
+   pip install -r requirements.txt
+   ```
+3. Run any of the applications in `python/bin/` or the example code in `python/examples/`. For example, to plot results
+   from a `*.p1log` file or recorded in an Atlas log:
+   ```bash
+   python3 bin/p1_display.py /path/to/log/directory
+   ```
+
+#### Virtual Environment
```

Comment:
While good advice I think this shouldn't be here it or be in an appendix. It's creates noise if you're just looking for information

Reply:
Sure, I'll move it to the bottom of the page